package.json

```diff
@@ -98,8 +98,8 @@
     "compile": "tsc -p . && cp src/*.json build/src && cp src/*.js build/src && cp -r test/fixtures build/test",
     "compile-protos": "pbjs -t json google/longrunning/operations.proto -p ./protos > protos/operations.json && pbjs -t json google/rpc/status.proto google/rpc/error_details.proto google/protobuf/any.proto -p ./protos > protos/status.json && pbjs -t static-module google/longrunning/operations.proto -p ./protos > protos/operations.js && pbts protos/operations.js -o protos/operations.d.ts",
```

Comment:
Should we compile iam_service protos similar to operation.proto?

Reply:
Good catch! `iam_service.json` needs to be in this command.

Reply:
Added another script to compile `compile-iam-protos` for IAM service.  thanks !

Reply:
@alexander-fenster seems `google/iam/v1/*.proto` does not work in the compile script. Now I'm explicitly calls every proto file, is there an efficient way to do that?